content/en/community/_index.md

```diff
@@ -10,41 +10,20 @@ aliases:
-
-* **Host the CHT?**
-
-  Checkout the [hosting section]({{% ref "hosting" %}})
-
-* **Get involved in an initiative?**
-
-   See the current 
-   [CHT related meetings](https://forum.communityhealthtoolkit.org/t/community-events/4641) and check the
-   [CHT Roadmap to view squad activity](https://github.com/orgs/medic/projects/112/views/24) 
-
-* **Contribute with translations?**
-
-  See our [translating guide]({{% ref "building/translations/managing" %}}).
-  Translators without developer backgrounds, open a 
-  [GitHub isssue](https://github.com/medic/cht-core/issues/new/choose) or post to the 
-  [forum](https://forum.communityhealthtoolkit.org) if you need help!
-
-* **Share your research, designs, or health practices?**
-
-   [Share your expertise](https://forum.communityhealthtoolkit.org) on the forum or sign up for the [newsletter](https://communityhealthtoolkit.org/contact) to get the latest updates.
+{{< callout emoji="👋" >}}
+  There are many ways to get involved! Do you want to...
+{{< /callout >}}
+
+{{< cards >}}
+  {{< card link="contributing/first-time-contributors" title="Contribute for the First Time?" subtitle="Start with the first time contributors guide" icon="heart" >}}
+  {{< card link="https://academy.communityhealthtoolkit.org/" title="Take Courses on the CHT?" icon="academic-cap" subtitle="Head over to the CHT Academy" >}}
+  {{< card link="contributing/" title="Contribute Code or Documentation" subtitle="Develop and contribute to the the CHT or help improve the documentation" icon="code" >}}
+  {{< card link="contributing/code/quality-assistance" title="Test the CHT?" subtitle="Help test and report bugs and participate in QA testing for new features" icon="document-text" >}}
+  {{< card link="../building" title="Build and Host a CHT App?" subtitle="Learn to build and host CHT apps and share your experience" icon="device-mobile" >}}
+  {{< card link="../designing" title="Explore CHT Design and UX?" subtitle="Explore design resources and contribute to UX improvements" icon="sparkles" >}}
+  {{< card link="../building/translations" title="Help Translating the CHT?" subtitle="Help make the CHT available in your language" icon="translate" >}}
+  {{< card link="https://forum.communityhealthtoolkit.org" title="Join the Forum Discussions?" subtitle="Participate in discussions, share research, suggest new functionalities and help answer questions about the CHT" icon="chat-alt-2" >}}
+  {{< card link="https://github.com/orgs/medic/projects/112/views/24" title="Get Involved in an Initiative?" subtitle="See the CHT Roadmap to view current squad activities" icon="user-group" >}}
+  {{< card link="https://calendar.google.com/calendar/u/0/r?cid=c_f39349b2aa6448b417e38084d5b7bb28bf3519fe588efff33e1ac638bd890620@group.calendar.google.com" title="Participate in a CHT Event?" subtitle="Subscribe to the CHT Calendar and never miss a community call" icon="calendar" >}}
```

Comment:
If you're not logged into google you get sent to the Google Calendar App page and it's not clear how to get to the real resource.  Maybe remove this feature for now and we can add it latter with a better link?

 ```suggestion
```

Reply:
I will link to the [Community Calls forum category](https://forum.communityhealthtoolkit.org/c/community/community-calls/33) instead; that could be equally helpful.